src/rss/filing_tracker.py

```
@@ -0,0 +1,290 @@
+class FilingTracker:
+    """Tracks all unique bankruptcy filings from RSS feeds"""
+    
+    def __init__(self, storage_file: Path = Path("all_bankruptcy_filings.json")):
+        """
+        Initialize filing tracker
+        
+        Args:
+            storage_file: Path to JSON file for persistent storage
+        """
+        self.storage_file = storage_file
+        self.filings: Dict[str, Dict[str, Any]] = {}
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)